builds/abi-compliance-checker/ci_build.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -x
+
+cd ../../
+
+mkdir tmp
+BUILD_PREFIX=$PWD/tmp
+
+CONFIG_OPTS=()
+CONFIG_OPTS+=("CFLAGS=-I${BUILD_PREFIX}/include -g -Og")
+CONFIG_OPTS+=("CPPFLAGS=-I${BUILD_PREFIX}/include")
+CONFIG_OPTS+=("CXXFLAGS=-I${BUILD_PREFIX}/include -g -Og")
+CONFIG_OPTS+=("LDFLAGS=-L${BUILD_PREFIX}/lib")
+CONFIG_OPTS+=("PKG_CONFIG_PATH=${BUILD_PREFIX}/lib/pkgconfig")
+CONFIG_OPTS+=("--prefix=${BUILD_PREFIX}")
+CONFIG_OPTS+=("--enable-drafts=no")
+
+function print_abi_api_breakages() {
+   echo "ABI breakages detected:"
+   cat compat_reports/libzmq/${LATEST_VERSION}_to_HEAD/abi_affected.txt | c++filt
+   echo "API breakages detected:"
+   cat compat_reports/libzmq/${LATEST_VERSION}_to_HEAD/src_affected.txt | c++filt
+   exit 1
+}
+
+./autogen.sh
+./configure "${CONFIG_OPTS[@]}"
+make VERBOSE=1 -j5
+abi-dumper src/.libs/libzmq.so -o ${BUILD_PREFIX}/libzmq.head.dump -lver HEAD
+
+git clone --depth 1 -b latest_release https://github.com/zeromq/libzmq.git latest_release
```

Comment:
Can you not do this from the existing clone?

Reply:
Travis does a depth 50 clone which means the last tag might not be there

Reply:
You can do: `git fetch --depth 1 origin latest_release:latest_release` even on a shallow clone, but now I'm not so sure it saves much. The `latest_release` branch is not really old enough to make a suitable test.

Reply:
sure, feel free to send a PR to update it